docs/.hooks/algolia.py

```diff
@@ -36,6 +36,10 @@ class AlgoliaRecord(TypedDict):
 
 
 def on_page_content(html: str, page: Page, config: Config, files: Files) -> str:
+    if not os.getenv('CI_DOCS_BUILD'):
+        # Only run in CI
+        return html
```

Comment:
Can we use this? https://github.blog/changelog/2020-04-15-github-actions-sets-the-ci-environment-variable-to-true/
```suggestion
    if not os.getenv('CI'):
        return html
```

Reply:
Yes makes sense

Reply:
I was only aware of `GITHUB_ACTIONS` and didn't want to tie the hook to GH